cime_config/config_grids.xml

```diff
@@ -4488,7 +4488,7 @@
 
     <gridmap ocn_grid="WC14to60E2r3" rof_grid="JRA025">
       <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_JRA025_to_WC14to60E2r3_smoothed.r150e300.200929.nc</map>
-      <map name="ROF2OCN_LIQ_RMAPNAME">cpl/cpl6/map_JRA025_to_WC14to60E2r2_smoothed.r150e300.200929.nc</map>
```

Comment:
@jonbob while not exactly a parameter setting, I fixed this here as it was a setting that was causing WC14 failure for JRA.  If you'd prefer me to separate this to a different PR, I'm happy to do so.  It wasn't clear to me if this was a clear cut different PR or not.

Reply:
Oh, maybe we should make it separate? Or else we can make this PR more for generalized EC30t60 and WC14 support, and include this and the pe-layout changes as well?

Reply:
Why don't we go with the latter and put in the pe layouts here instead of a ton of extremely small PRs.

Reply:
OK, we can change the title of this PR to reflect general support. I'll work on adding the layout update here today, and maybe we can get 4091 in first?

Reply:
Sounds like a good plan to me